src/assets/styles/styled.ts

```diff
@@ -0,0 +1,268 @@
+    max-height: ${(props) => props.maxHeight || 'none'};
+    min-height: ${(props) => props.minHeight || 'auto'};
+    flex-direction: ${(props) =>
+      props.flexDir ? (typeof props.flexDir !== 'string' ? props.flexDir[1] : props.flexDir) : 'row'};
+    flex-wrap: ${(props) =>
+      props.flexWrap
+        ? (typeof props.flexWrap === 'string' && props.flexWrap === 'row-reverse') ||
+          (typeof props.flexWrap === 'string' && props.flexWrap === 'row') ||
+          (typeof props.flexWrap !== 'string' && props.flexWrap[1] === 'row-reverse') ||
+          (typeof props.flexWrap !== 'string' && props.flexWrap[1] === 'row')
+          ? 'wrap'
+          : 'nowrap'
+        : props.flexDir
+        ? (typeof props.flexDir === 'string' && props.flexDir === 'column') ||
+          (typeof props.flexDir !== 'string' && props.flexDir[1] === 'column')
+          ? 'nowrap'
+          : 'wrap'
+        : 'wrap'};
+    align-items: ${(props) =>
+      props.alignItems
+        ? typeof props.alignItems !== 'string'
+          ? props.alignItems[1]
+          : props.alignItems
+        : 'flex-start'};
+    justify-content: ${(props) =>
+      props.justifyCont
+        ? typeof props.justifyCont !== 'string'
+          ? props.justifyCont[1]
+          : props.justifyCont
+        : 'flex-start'};
+    box-shadow: ${(props) => props.bs || 'none'};
+    background-color: ${(props) =>
+      props.bgColor ? (typeof props.bgColor !== 'string' ? props.bgColor[1] : props.bgColor) : 'transparent'};
+    flex-grow: ${(props) => props.grow || '0'};
+    border-radius: ${(props) => (props.br ? props.br / 16 : 0)}rem;
+    ${(props) =>
+      props.borders
+        ? typeof props.borders !== 'string'
+          ? props.borders.length > 1
+            ? props.borders[1].map((item: any) => `border-${item.side}: 1px solid ${item.color || '#000'};`)
+            : props.borders.map((item: any) => `border-${item.side}: 1px solid ${item.color || '#000'};`)
+          : ` ${props.borders ? props.borders : props.theme.appBody.border};`
+        : 'border: 0;'}
+  }
+`
+
+export const Row = styled.div<Partial<CSS.Properties & WrapProps>>`
+  display: flex;
+  flex-wrap: wrap;
+  flex-grow: 1;
+  align-items: ${(props) => props.alignItems};
+  justify-content: ${(props) => props.justifyContent || 'flex-start'};
+  margin: ${(props) => props.margin || '0 -0.625rem'};
+`
+
+export const Column = styled.div<Partial<CSS.Properties & ColumnProps>>`
+  padding: ${(props) => props.padding || '0 0.625rem'};
+  max-width: ${(props) => (props.xs !== 'hidden' && props.xs ? props.xs + '%' : '100%')};
+  flex: 0 0 ${(props) => (props.xs !== 'hidden' && props.xs ? props.xs + '%' : '100%')};
+  ${(props) => props.flexCol && 'display:flex;flex-direction:column;'}
+  ${(props) => props.flexRow && 'display:flex;flex-wrap:wrap; align-items:center'}
+  ${(props) => props.stretch && 'height:100%;'}
+  @media (max-width:767px) {
```

Comment:
For this we usually have these media queries https://github.com/gnosis/cowswap/blob/b366fa833887f6a86dacf93f425e2dc6f250c811/src/theme/styled.ts#L70

Reply:
yes please review the theme inside custom/theme and src/theme

Reply:
Commented elsewhere as well. We use this format:

```
${({ theme }) => theme.mediaWidth.upToSmall`
  // stuff
`};
```

Reply:
done